docs/releases/1.11.txt

```diff
@@ -205,7 +205,9 @@ Database backend API
 Miscellaneous
 -------------
 
-* ...
+* The :meth:`SyndicationFeed.latest_post_date <django.utils.feedgenerator.SyndicationFeed.latest_post_date>`
```

Comment:
please wrap docs at 79 chars. did you see my comment in the other thread about adding an additional test to make sure this works with `USE_TZ=True` and `False`?